broker/aws/impl.go

```diff
@@ -423,33 +435,33 @@ func (b *Broker) getConsoleURLForAccountRole(accountName string, roleName string
 	if err != nil {
 		return "", err
 	}
-	awsDestinationURL := "https://console.aws.amazon.com/"
-	targetUrl := fmt.Sprintf("https://signin.aws.amazon.com/federation?Action=login&Issuer=https://example.com&Destination=%s&SigninToken=%s", awsDestinationURL, tokenOutput.SigninToken)
+	targetUrl := fmt.Sprintf("%s?Action=login&Issuer=https://example.com&Destination=%s&SigninToken=%s", federationUrl, awsDestinationURL, tokenOutput.SigninToken)
 
 	b.logger.Debugf(1, "targetURL=%s", targetUrl)
-	//TODO check region from account config
-	//region := "us-east-1"
 
-	//return "", errors.New("Not implemented")
 	return targetUrl, nil
 }
 
 func (b *Broker) generateTokenCredentials(accountName string, roleName string, userName string) (*broker.AWSCredentialsJSON, error) {
-	assumeRoleOutput, err := b.withProfileAssumeRole(accountName, masterAWSProfileName, roleName, userName)
+	assumeRoleOutput, region, err := b.withProfileAssumeRole(accountName, masterAWSProfileName, roleName, userName)
 	if err != nil {
 		b.logger.Debugf(1, "cannot assume role for account %s with master account, err=%s ", accountName, err)
 		// try using a direct role if possible then
-		assumeRoleOutput, err = b.withProfileAssumeRole(accountName, accountName, roleName, userName)
+		assumeRoleOutput, region, err = b.withProfileAssumeRole(accountName, accountName, roleName, userName)
 		if err != nil {
 			b.logger.Printf("cannot assume role for account %s, err=%s", accountName, err)
 			return nil, err
 		}
 	}
 	b.logger.Debugf(2, "assume role success for account=%s, roleoutput=%v", accountName, assumeRoleOutput)
+	if region != "us-gov-west-1" {
```

Comment:
Please change this test to: `if !strings.HasPrefix(region, "us-gov-")` so that other GovCloud regions are supported.